angular/src/lib/tooltip/tooltip.component.ts

```diff
@@ -16,6 +22,10 @@ export class TooltipComponent implements OnInit {
   @Input() text: string = '';
   @Input() tooltipTemplate: TemplateRef<any>;
   @Input() id: string;
+  @Input() maxWidth: number;
+
+  @HostBinding('style.max-width.px')
```

Comment:
@daanishnasir If you have HostBinding do you need the property on the div as well?